lib/livebook_web/live/user_component.ex

```diff
@@ -41,6 +41,12 @@ defmodule LivebookWeb.UserComponent do
             spellcheck="false"
             disabled={Livebook.Config.identity_readonly?()}
           />
+          <.text_field
+            field={f[:email]}
```

Comment:
@josevalim perhaps email only makes sense in case of ZTA, or do we still want to make it editable?

Reply:
I posted above. We don't want to make it editable. Only from ZTA. :)